doc/source/rllib/rllib-training.rst

```diff
@@ -1163,45 +1163,49 @@ calls an "evaluation step" is run:
     }
 
 
-One such evaluation step runs over ``evaluation_duration`` episodes or timesteps, depending
+An evaluation step runs - using its own RolloutWorkers - for ``evaluation_duration`` episodes or timesteps, depending
 on the ``evaluation_duration_unit`` setting, which can be either "episodes" (default) or "timesteps".
 
 
 .. code-block:: python
 
-    # Every time we do run an evaluation step, run it for exactly 10 episodes.
+    # Every time we run an evaluation step, run it for exactly 10 episodes.
     {
         "evaluation_duration": 10,
         "evaluation_duration_unit": "episodes",
     }
-    # Every time we do run an evaluation step, run it for close to 200 timesteps.
+    # Every time we run an evaluation step, run it for (close to) 200 timesteps.
     {
         "evaluation_duration": 200,
         "evaluation_duration_unit": "timesteps",
     }
 
 
+Note: When using ``evaluation_duration_unit=timesteps`` and your ``evaluation_duration`` setting is NOT dividable
+by the number of evaluation workers (configurable via ``evaluation_num_workers``), RLlib will run a slightly different number of
+timesteps during each evaluation step.
+
 Before each evaluation step, weights from the main model are synchronized to all evaluation workers.
 
-Normally, the evaluation step is run right after the respective train step. For example, for
-``evaluation_interval=2``, the sequence of steps is: ``train, train, eval, train, train, eval, ...``.
+By default, the evaluation step is run right after the respective training step. For example, for
+``evaluation_interval=2``, the sequence of events is: ``train, train, eval, train, train, eval, ...``.
 For ``evaluation_interval=1``, the sequence is: ``train, eval, train, eval, ...``.
 
 However, it is possible to run evaluation in parallel to training via the ``evaluation_parallel_to_training=True``
-config setting. In this case, both steps (train and eval) are run at the same time via threading.
+config setting. In this case, both training- and evaluation steps are run at the same time via threading.
 This can speed up the evaluation process significantly, but leads to a 1-iteration delay between reported
-training results and evaluation results (the evaluation results are behind b/c they use slightly outdated
-model weights).
+training- and evaluation results. The evaluation results are behind b/c they use slightly outdated
```

Comment:
in the case of parallel eval, are the eval metrics still reported at the timestep which they were originally launched from?

Reply:
Nope, we can't report retrospectively. Eval results are reported together with the train step (even if evaluation workers operate on policies that are 1 train step behind, weights-wise).